build.gradle.kts

```diff
@@ -175,15 +180,33 @@ publishing {
     }
 }
 
-signing {
-    setRequired({
-        gradle.taskGraph.hasTask("publish")
-    })
-    sign(publishing.publications["mavenJava"])
+    repositories {
+        maven {
+            url = uri("$artifactoryContextUrl/libs-release-local")
+            credentials {
+                username = artifactoryUser
+                password = artifactoryPassword
+            }
+        }
+    }
```

Comment:
This repositories section isn't really needed because you don't need to pull artifacts from the artifactory.

Reply:
No negative effect though. It will just fallback to the artifactory in case something's not available in mavenCentral.